core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.html

```diff
@@ -9,6 +9,7 @@ <h3 *ngIf="!editingTitle && formTitle" class="texera-workspace-property-editor-t
     nzSize="small"
     nzTooltipPlacement="bottom"
     nzType="text"
+    [disabled]="!interactive"
```

Comment:
so seems we have to disable buttons one by one. See if there is a way to disable the parent element (e.g., the form) entirely rather that individual component on the form? I think if so it's easier to maintain.

Reply:
Yes, we can place all the elements under one single `div` and use CSS's `pointer-events` to disable all interactivity. But I do not feel this is user-friendly because user cannot see the buttons disabled (grayed out) this way. And I have not found a way that's easier to maintain and make the buttons still be `disabled`.

Reply:
Right makes sense, let's just manage each button individually